versioned_docs/version-1.21.1/resources/server/datamaps/builtin.md

```diff
@@ -224,5 +224,31 @@ Example:
 }
 ```
 
+
+## `neoforge:villager_types`
+
+Allows configuring the villager type that will spawn based on its biome, as a replacement for `VillagerType#BY_BIOME` (which will be ignored in 1.22). It is located at `neoforge/data_maps/worldgen/biome/villager_types.json` and its objects have the following structure:
+
+```json5
+{
+    // The villager type that will spawn in this biome
+    "villager_type": "minecraft:desert"
+    //  If a biome is not present the default villager type of `minecraft:plains` will be used
```

Comment:
This should go above the example and underneath the previous comment.

As for the wording, I would phrase it more like 'If no villager type is specified for a biome, then 'Minecraft:plains' will be used.